README.md

```diff
@@ -653,7 +680,36 @@ A more exhaustive enumeration of `react-hotkeys` behaviour can be found by revie
 
 `react-hotkeys` provides the `getApplicationKeyMap()` function for getting a mapping of all actions and key sequences that have been defined by components that are currently mounted.
 
-They are returned as an object, with the action names as keys (it is up to you to decide how to translate them to be displayed) and arrays of key sequences that trigger them, as keys.
+They are returned as an object, with the action names as keys, and the values are objects describing the key map.
+
+Regardless of which syntax you used to define the keymap, they always appear in the following syntax:
+
+```
+{
+  ...
```

Comment:
⬜️ ✏️Comment out the ellipses and place them at the end, to indicate further attributes or array elements.